docs/content/specs-defs/includes/bicep/shared/non-functional/BCPNFR1.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: BCPNFR1 - Data Types
+title: BCPNFR1 - User-defined types - General
```

Comment:
I think this one should temporarily be a SHOULD and not a MUST

Reply:
Do we have temporary specs? :D I phrased it in the way I did with the perspective of 'how it should be' or what we know what 1.0 will be. I understand why you want to SHOULD, but I guess I wasn't aware we're writing specs in this way (but instead don't enforce them all before 1.0)